app/views/early_years_payment/start/claims/check_your_email.html.erb

```diff
@@ -0,0 +1,35 @@
+<% content_for(:page_title, page_title(t("early_years_payment_start.check_your_email_page.title"), journey: current_journey_routing_name)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-notification-banner govuk-notification-banner--success" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+    <div class="govuk-notification-banner__header">
+      <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+        Success
+      </h2>
+    </div>
+    <div class="govuk-notification-banner__content">
+      <h3 class="govuk-notification-banner__heading">
+        Email sent to <%= journey_session.answers.email_address %>
+      </h3>
+    </div>
+  </div>
```

Comment:
theres a component which should reduce the amount of markup you need see https://govuk-components.netlify.app/components/notification-banner/#notification-banner-with-success

Reply:
i'd probably also move this one line down so its nested in `govuk-grid-column-two-thirds` as typically all the content is two thirds, unless its been explicitly designed for full width?

Reply:
I'm basing the banner on the prototype (https://dfedigital.atlassian.net.mcas.ms/browse/LUPEYALPHA-762) which has it full width.